src/memgraph.cpp

```diff
@@ -84,11 +90,14 @@ void InitFromCypherlFile(memgraph::query::InterpreterContext &ctx, memgraph::dbm
   while (std::getline(file, line)) {
     if (!line.empty()) {
       try {
+        // TODO remove security issue
+        spdlog::trace("Executing line: {}", line);
         auto results = interpreter.Prepare(line, {}, {});
         memgraph::query::DiscardValueResultStream stream;
         interpreter.Pull(&stream, {}, results.qid);
-      } catch (const memgraph::query::UserAlreadyExistsException &e) {
-        spdlog::warn("{} The rest of the init-file will be run.", e.what());
+      } catch (std::exception const &e) {
+        spdlog::warn("Exception occurred while executing one line. The rest of the init-file will be run. {}",
+                     e.what());
       }
```

Comment:
This is suspicious. I think I did the same try-catch in one PR but @andrejtonev had some comment that we are catching things we shouldn't. Here if you catch exception, you will continue to execute other lines but is that expected behaviour? Not sue to me it would also seem logical to abort any other commands once the failure occurs

Reply:
Try catch was here before if you check diff, but only for specific exceptions. Now we can throw some of the coordinator exceptions. I can make it specific for only coord exceptions. In my opinion it makes sense to abort completely but I didn't want to disrupt logic from before, don't know if some of the customers are relying on it

Reply:
Check this PR https://github.com/memgraph/memgraph/pull/1465

Reply:
comments from pr